playground/frontend/Dockerfile

```diff
@@ -23,8 +23,13 @@ ARG FLUTTER_VERSION=3.7.3
 # Setting up.
 # ------------------------------------------------
 
+COPY . /app
+WORKDIR /app
+
 RUN apt-get update && apt-get install -y wget xz-utils git
 
+RUN cat /app/playground_components/lib/src/build_metadata.g.dart
```

Comment:
Remove

Reply:
I would rather keep it in build logs. If anything goes wrong with generation, it will be problematic to extract this file content.

Reply:
Deleted.